src/components/SliderIMDB/SliderIMDB.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './styles/SliderIMDB.scss';
+import type { SliderProps } from 'react-aria-components';
+import {
+  Slider,
+  Label,
+  SliderOutput,
+  SliderTrack,
+  SliderThumb
+} from 'react-aria-components';
+
+interface SliderIMDBProps<T> extends SliderProps<T> {
+  label?: string;
+  thumbLabels?: string[];
+}
+
+export function SliderIMDB<T extends number | number[]>({
+  label,
+  thumbLabels,
+  ...props
+}: SliderIMDBProps<T>) {
+  return (
+    <Slider {...props} minValue={0} maxValue={10} step={0.1}>
```

Comment:
add default props for this attributes like:
({
label,
thumbLabels,
minValue = 0,
maxValue = 0,
step = 0.1
})